src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,66 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static List<String> delimeter = new ArrayList<>(List.of(":", ","));
+
+    public static int calculate(String input) {
+        if (isBlank(input)) {
+            return 0;
+        }
+
+        int[] numbers = parseInput(input);
+
+        return sum(numbers);
```

Comment:
```suggestion
        return sum(parseInput(input));
```
로컬 변수 없이 구현해도 되지 않을까?